packages/prebuild-config/src/plugins/unversioned/expo-splash-screen/getAndroidSplashConfig.ts

```diff
@@ -0,0 +1,72 @@
+import { ExpoConfig } from '@expo/config-types';
+
+export type SplashScreenConfig = {
+  xxxhdpi: string | null;
+  xxhdpi: string | null;
+  xhdpi: string | null;
+  hdpi: string | null;
+  mdpi: string | null;
+  backgroundColor: string | null;
+  resizeMode: 'contain' | 'cover' | 'native';
+};
+
+const defaultResizeMode = 'contain';
+
+export function getAndroidSplashConfig(
+  config: Pick<ExpoConfig, 'splash' | 'android'>
+): SplashScreenConfig | null {
+  // Respect the splash screen object, don't mix and match across different splash screen objects
+  // in case the user wants the top level splash to apply to every platform except android.
+  if (config.android?.splash) {
+    const splash = config.android?.splash;
+    return {
+      // TODO: Does this not support `image`?
```

Comment:
it kinda does, i believe, but if you provide an image that is too large it could crash the app on low end devices